YubiKit/YubiKit/Sessions/Shared/Services/MGMT/YKFKeyMGMTService.m

```diff
@@ -63,15 +63,22 @@ - (void)readConfigurationWithCompletion:(YKFKeyMGMTServiceReadCompletionBlock)co
             completion(nil, [YKFKeyMGMTError errorWithCode:YKFKeyMGMTErrorCodeUnexpectedResponse]);
             return;
         }
+        
+        if (selectionResponse.version.major < 5) {
```

Comment:
You don't need to check version here...we handle that applet might not be available by the error that returned on selection.
And BTW the error that applet doesn't not exist on YubiKey is not unexpected.
Selection of applet on NEO actually returns YKFKeyAPDUErrorCodeInsNotSupported. While we expect YKFKeyAPDUErrorCodeMissingFile. You need just to swap this error codes...

Reply:
I didn't realize that NEO returns smth different from what expected.